src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1703,6 +1703,22 @@
         JavaLanguageLexer.LITERAL_DOUBLE;
 
     /**
+     * <p>For example:</p>
```

Comment:
Move example after explanation as it is done in other javadoc

Reply:
Done with changes @romani